include/ghex/communication_object.hpp

```diff
@@ -365,15 +364,15 @@ class communication_object
         using test_t = pattern_container<grid_type, domain_id_type>;
         std::map<const test_t*, int> pat_ptr_map;
         int                          max_tag = 0;
-        for_each(iter_pairs_t, [&pat_ptr_map, &max_tag](auto iter_pair) {
+        for_each(iter_pairs_t, [&pat_ptr_map, &max_tag](std::size_t, auto iter_pair) {
```

Comment:
What is happening here, why is a size_t appearing in the lambda, but not used?

Reply:
The for each was changed to also pass an iteration count as first argument to the user supplied visitor function.